prow/plugins/label/label_test.go

```diff
@@ -575,6 +575,16 @@ func TestLabel(t *testing.T) {
 			expectedBotComment:    false,
 			action:                github.GenericCommentActionDeleted,
 		},
+		{
+			name:                  "Strip markdown comments",
+			body:                  "<!--\n/kind bug\n/kind cleanup\n-->\n/area infra",
+			repoLabels:            []string{"area/infra"},
+			issueLabels:           []string{},
+			expectedNewLabels:     formatLabels("area/infra"),
+			expectedRemovedLabels: []string{},
+			commenter:             orgMember,
+			action:                github.GenericCommentActionCreated,
+		},
 	}
```

Comment:
add a test that has multiple comments and make sure this ignores commented content but still recognizes uncommented content:

```
<!--
/area comment1
-->
/area infra1
<!--
/area comment2
-->
/area infra2
```

Reply:
Added the test as suggested.